### PR TITLE
disabled macOSX default mecab library

### PR DESCRIPTION
**Pull request by myoan**
hi masahiro!
I fixed code which you requested.
please merge this.

### CODE REVIEW

**Comment by myoan** [comment]
sorry, but I fixed bug mecab new method.
please merge this.

File: package/mecab/mecab_glue.c [hunk 1]
```diff
@@ -22,16 +22,397 @@
...
+{
+	struct _kMecabNode *node = (struct _kMecabNode*)sfp[0].asObject;
+	mecab_node_t* prev = node->node->prev;
+	struct _kMecabNode* ret;
+	if (node != NULL) {
+		ret = (struct _kMecabNode*)KLIB new_kObject(kctx, O_ct(sfp[K_RTNIDX].asObject), 0);
+		ret->node = prev;
+	}
+	else {
+		ret = NULL;
```
Comment (imasahiro) [inline package/mecab/mecab_glue.c]: You do not set NULL.
please use K_NULL or RETURN_DefaultObjectValue();

File: package/mecab/mecab_glue.c [hunk 2]
```diff
@@ -22,16 +22,397 @@
...
+{
+	struct _kMecabNode *node = (struct _kMecabNode*)sfp[0].asObject;
+	mecab_node_t* enext = node->node->enext;
+	struct _kMecabNode* ret;
+	if (node != NULL) {
+		ret = (struct _kMecabNode*)KLIB new_kObject(kctx, O_ct(sfp[K_RTNIDX].asObject), 0);
+		ret->node = enext;
+	}
+	else {
+		ret = NULL;
```
Comment (imasahiro) [inline package/mecab/mecab_glue.c]: You do not set NULL.
please use K_NULL or RETURN_DefaultObjectValue();

**Comment by myoan** [comment]
fix NULL return

**Pull request merged**